schemas/yaml/Orthography.yml

```
@@ -144,6 +144,12 @@ properties:
     format: uri
     description: 'A URL where this Orthography may be accessed in JSON format'
 
+  writingSystem:
+    title: Writing System
+    type: string
+    pattern: '^[A-Z]{1}[a-z]{3}[0-9]{3}$'
```

```suggestion
    pattern: '^[A-Z]{1}[a-z]{3}$'
```

My intention was that we'd only use the 4-letter code and not the number. (The two codes can be used independently of each other.) Forgot to say that though. 😬